Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Projection/NodeRecord.php

```diff
@@ -118,7 +119,7 @@ public function removeFromDatabase(Connection $databaseConnection, string $table
     public static function fromDatabaseRow(array $databaseRow): self
     {
         return new self(
-            NodeRelationAnchorPoint::fromString($databaseRow['relationanchorpoint']),
+            NodeRelationAnchorPoint::fromInteger((int)$databaseRow['relationanchorpoint']),
```

Comment:
Why is the explict cast type needed here but not on other places?

Reply:
this generic constructor is used from a couple of places and I couldn't be sure we would get integers everywhere. But I can see if I can do something about it.